pandas/tests/groupby/test_groupby.py

```diff
@@ -249,11 +249,11 @@ def test_len():
 
 def test_basic_regression():
     # regression
-    T = [1.0 * x for x in lrange(1, 10) * 10][:1095]
-    result = Series(T, lrange(0, len(T)))
+    T = [1.0 * x for x in list(range(1, 10)) * 10][:1095]
```

Comment:
I think the `[:1095]` can be removed since `T` should have length 90 making the slice unnecessary.